instantsearch/src/main/java/com/algolia/instantsearch/helpers/InstantSearch.java

```diff
@@ -398,9 +465,9 @@ private void prepareWidget(@Nullable View rootView, View widget, @Nullable List<
                 // Link hits to activity's empty view //TODO: Remove rootView parameter if getRootView always works
                 ((Hits) widget).setEmptyView(getEmptyView(widget.getRootView()));
 
-                itemLayoutId = ((Hits) widget).getLayoutId();
+                int itemLayoutId = ((Hits) widget).getLayoutId();
 
-                if (itemLayoutId == -42) {
+                if (itemLayoutId == 0) {
```

Comment:
Why the change ?

Reply:
0 is not a valid`ResId` value, hence no need for a magic number